parser/src/types.rs

```diff
@@ -943,6 +943,12 @@ impl<'input> Member<'input> {
         self.bit_offset
     }
 
+    /// Return true if this member has the DW_AT_bit_size attribute
+    #[inline]
+    pub fn bit_size_attr_present(&self) -> bool {
```

Comment:
```suggestion
    /// Return true if this member defines a bit size instead of using the size of the type.
    #[inline]
    pub fn is_bit_field(&self) -> bool {
```
While this crate currently only supports DWARF, that goal was that it might be able to support PDB as well, so let's try to leave DWARF details out of the API.

Reply:
Thank you for the suggestion. Suggestion applied now. Hope you don't mind that I force-pushed.